test/integration/providers/p4runtime.go

```diff
@@ -7,39 +7,47 @@ import (
 	"context"
 	"fmt"
 	"github.com/antoninbas/p4runtime-go-client/pkg/client"
+	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials/insecure"
 	"time"
-
-	p4_v1 "github.com/p4lang/p4runtime/go/p4/v1"
 )
 
 var (
 	stopCh   chan struct{}
 	grpcConn *grpc.ClientConn
+
+	// DefaultElectionID use reader election ID so that pfcpiface doesn't lose mastership.
+	DefaultElectionID = p4_v1.Uint128{High: 0, Low: 1}
 )
 
-func ConnectP4rt(addr string, electionID p4_v1.Uint128) (*client.Client, error) {
+func ConnectP4rt(addr string, asMaster bool) (*client.Client, error) {
 	grpcConn, err := grpc.Dial(addr, grpc.WithTransportCredentials(insecure.NewCredentials()))
 	if err != nil {
 		return nil, err
 	}
 
 	c := p4_v1.NewP4RuntimeClient(grpcConn)
 
-	stopCh = make(chan struct{})
+	p4RtC := client.NewClient(c, 1, DefaultElectionID, client.DisableCanonicalBytestrings)
```

Comment:
Doesn't (re-)using the `DefaultElectionID` prevent us from **ever** re-connecting to the same server again?

Reply:
I just realized that these function cannot be called concurrently because of the shared global state anyway. Might not be a problem then.